scripts/fetch_and_build_rpm

```diff
@@ -172,34 +172,36 @@ sed -i -E -e "\\|^Source0:|s|https:.*|${tarballurl}|" \
           -e "/^%global pgpackageversion/s/[0-9.]+$/${PGVERSION}/" \
     "${builddir}/${pkgname}.spec"
 
-osname=$(awk '{print $1}' /etc/system-release)
-if [ "${osname}" == 'Oracle' ]; then
+os_name=$(awk '{print $1}' /etc/system-release)
+os_release=$(awk '{print $4}' /etc/system-release)
+if [ "${os_name}" == 'Oracle' ]; then
     locale='C'
-elif [ "${osname}" == 'Fedora' ] || [ "${osname}" == 'CentOS' ]; then
+elif [ "${os_name}" == 'Fedora' ] || [[ "${os_name}" == 'CentOS'  && "${os_release}" == 8* ]]; then
     locale='C.utf8'
 else
     locale='en_US.utf8'
 fi
 
+if [ "${versioning}" == 'fancy' ]; then
+  infix=$(echo "${packageversion}" | grep -oE '^[0-9]+\.[0-9]+' | tr -d '.')
+  if [ "${release_type}" == "stable" ]; then
+      package_prefix="${infix}"
+  else
+      package_prefix="${infix}_${release_type}"
+  fi
+  sed -i -E "1i %global pkginfix ${package_prefix}" "${builddir}/${pkgname}.spec"
+fi
+
 case "${1}" in
     release)
         # add minor/major version to package name if using fancy versioning
```

Comment:
This comment no longer makes sense. Can we update that with a simple explanation on why we do not have any steps for release?